tracing-subscriber/src/fmt/fmt_layer.rs

```diff
@@ -437,6 +478,20 @@ where
                 extensions.insert(fmt_fields);
             }
         }
+
+        if self.fmt_span.fmt_timing
+            && self.fmt_span.trace_close()
+            && extensions.get_mut::<Timings>().is_none()
+        {
+            extensions.insert(Timings::new());
+        }
+
+        if self.fmt_span.trace_new() {
+            event_from_span!(event = id, span, message = "new");
+            drop(extensions);
+            drop(span);
```

Comment:
The compiler will scream at you if you remove it because `span` is borrowed while `on_event` takes ownership of it. So github readers would benefit from a comment while IDE-users should see it quickly. I tried returning `event` to a different context, but Rust cannot express that (hence the macro, a function wouldn’t work).